### ISSUE TITLE
Translations not found

### ISSUE THREAD

**Issue by dolenec** [skodaconnect/homeassistant-skodaconnect#192]
**Describe the bug**
The problem is that Skoda Connect sensor do not have options to be translated in frontend neither in backend translation.

Missing translations:
minutes
days
Locked
Unlocked

Which entities:
lock.skoda_*_trunk_locked
lock.skoda_*_door_locked
sensor.skoda_*_parking_heater_heating_ventilation_duration
sensor.skoda_*_service_inspection_days
sensor.skoda_*_oil_inspection_days
...

So in translation tool (lokalise) we can not contribute translation as string is missing and allways showing English translation.

**Comment by Farfar** [comment]
Correct, no efforts have been made to incorporate support for translations.
I will add as an enhancement for future releases.

**Comment by MasterOfPuppets111** [comment]
Success has also a missing translation from entity:
request_results

**Comment by WebSpider** [comment]
This integration is no longer under active development. Please consider switching to https://github.com/skodaconnect/homeassistant-myskoda
